Lib/fontTools/feaLib/parser.py

```diff
@@ -1339,14 +1363,24 @@ def expect_string_(self):
             return self.cur_token_
         raise FeatureLibError("Expected a string", self.cur_token_location_)
 
-    def advance_lexer_(self):
-        self.cur_token_type_, self.cur_token_, self.cur_token_location_ = (
-            self.next_token_type_, self.next_token_, self.next_token_location_)
-        try:
-            (self.next_token_type_, self.next_token_,
-             self.next_token_location_) = self.lexer_.next()
-        except StopIteration:
-            self.next_token_type_, self.next_token_ = (None, None)
+    def advance_lexer_(self, comments = False):
+        if not self.ignore_comments and comments and len(self.cur_comments_):
+            self.cur_token_type_ = Lexer.COMMENT
+            self.cur_token_, self.cur_token_location_ = self.cur_comments_.pop(0)
+            return
+        else 
```

Comment:
python colons. Syntactic sugar that should be optional, especially designed to frustrate just me! :)